### PR TITLE
Feature/min balance community creation

### PR DESCRIPTION
**Pull request by lucca65**
## What issue does this PR close
Closes N/A. Depends on #396 

## Changes Proposed ( a list of new changes introduced by this PR)
- Add ability to specify the community min balance during community creation

## How to test ( a list of instructions on how to test this PR)
- Open `/community/new`
- Fill in all the fields, including the new min balance field
- Create community
- open [The block explorer](https://local.bloks.io/account/cambiatus.tk?loadContract=true&tab=Tables&table=stat&account=cambiatus.tk&scope=MIN&limit=100&nodeUrl=staging.cambiatus.io&coreSymbol=SYS&systemDomain=eosio) and check if the min balance is correct

### CODE REVIEW

File: src/elm/Page/Community/Editor.elm [hunk 1]
```diff
@@ -140,6 +142,7 @@ editForm community =
     , logoList = logoList
     , inviterReward = community.inviterReward
     , invitedReward = community.invitedReward
+    , minBalance = community.minBalance |> Maybe.withDefault -100
```
Comment (amiskov) [inline src/elm/Page/Community/Editor.elm:145]: Here we have `-100` as the default value, but for the `newForm` we have `-10`.

Maybe we can do this?
```elm
, minBalance = community.minBalance |> Maybe.withDefault newForm.minBalance
```

File: src/elm/Page/Community/Settings/Features.elm [hunk 2]
```diff
@@ -298,6 +298,7 @@ saveFeature feature state authorization { shared, accountName } community =
             , description = community.description
             , inviterReward = community.inviterReward
             , invitedReward = community.invitedReward
+            , minBalance = community.minBalance |> Maybe.withDefault -100
```
Comment (amiskov) [inline src/elm/Page/Community/Settings/Features.elm:301]: Same here: `-100` is used as the default value but in the form for the new community `-10` is used.

**Review by amiskov — CHANGES_REQUESTED** [review]
Spotted a couple of issues that are hardly possible but I think it worth adjusting them.

File: src/elm/Page/Community/Editor.elm (hunk 1 above)
Comment (lucca65) [reply src/elm/Page/Community/Editor.elm:145]: Done, this is way better

File: src/elm/Page/Community/Settings/Features.elm (hunk 2 above)
Comment (lucca65) [reply src/elm/Page/Community/Settings/Features.elm:301]: I'll change the default on the form and keep it like this

**Review by amiskov — APPROVED** [review]

**Pull request merged**